### ISSUE TITLE
Why is batch norm switched off?

### ISSUE THREAD

**Issue by alivaramesh** [Andrew-Brown1/Smooth_AP#15]
Can you please explain why batch-norm is turned off? does it have adverse effect when switched on? 
Thanks.

**Comment by Andrew-Brown1** [comment]
Hey,

It is true that the batch-norm layer is frozen during training on these datasets. In fact, in Deep Metric Learning we are fine-tuning models that are pre-trained on ImageNet, and so it is fairly standard to then freeze the batch norm layers during the fine-tuning stage. 
We did not experiment with training the batch-norm layers, so I do not know if it would have an adverse effect
